src/System.IO.UnmanagedMemoryStream/tests/UmaTests.cs

```diff
@@ -23,6 +23,13 @@ public static void UmaInvalidReadWrite()
                 Assert.Throws<ArgumentOutOfRangeException>(() => uma.Write(capacity, 12345));
                 Assert.Throws<ArgumentException>(() => uma.Write(capacity - 1, 0.123));
 
+                // UMA should throw when reading bad decimal values (some bits of flags are reserved and must be 0)
+                uma.Write(0, 0); // lo
```

Comment:
@stephentoub Yep, my bad. I forgot to run MSBuild from the tests directory, sorry about that.

As for the tests, I've fixed the null refs by using a `TestSafeBuffer` instead of a fake one.